src/blog/v8-release-84.md

```diff
@@ -0,0 +1,88 @@
+
+## JavaScript
+
+### Weak references and finalizers
+
+:::note
+**Warning!** Weak references and finalizers are advanced features! They depend on garbage collection behavior. Garbage collection is non-deterministic and may not occur at all.
+:::
+
+JavaScript is a garbage collected language, which means memory occupied by objects that are no longer reachable by the program may be automatically reclaimed when the garbage collector runs. With the exception of references in `WeakMap` and `WeakSet`, all references in JavaScript are strong and prevent the referenced object from being garbage collected. For instance,
+
+```javascript
+let globalRef = { callback: function() { console.log(“foo”); } };
+// As long as globalRef is reachable through the global scope,
+// neither it nor the function in its callback property will be collected.
+```
+
+JavaScript programmers can now hold on to objects weakly via the `WeakRef` feature. Objects that are referenced by weak references do not prevent their being garbage collected if they are not also strongly referenced.
+
+```javascript
+const globalWeakRef = new WeakRef({ callback: function() { console.log(“foo”); });
+(async function() {
+  globalWeakRef.deref().callback();
+  // Logs “foo” to console. Weak ref guaranteed to be alive for the first turn
+  // of the event loop when it is created.
+
+  await (new Promise((resolve, reject) => {
+    setTimeout(() => { resolve('foo'); }, 42);
+  });
+  // Wait for a turn of the event loop.
+
+  globalWeakRef.deref()?.callback();
+  // The object inside globalWeakRef may be garbage collected
+  // after the first turn since it is not otherwise reachable.
+})();
+```
+
+The companion feature to `WeakRef`s is `FinalizationRegistry`, which lets programmers register callbacks to be invoked after an object is garbage collected. For example, the program below may log “42” to the console after the unreachable object in the IIFE is collected.
+
+```javascript
+const registry = new FinalizationRegistry((heldValue) => {
+  console.log(holdings);
+});
+(function () {
+  let garbage = {};
+  registry.register(garbage, 42);
+  // The second argument is the “held” value which gets passed
+  // to the finalizer when the first argument is garbage collected.
+})();
+```
+
+Finalizers are scheduled to run on the event loop and never interrupt synchronous JavaScript execution.
+
+These are advanced and powerful features, and with any luck, your program won’t need them. Please see our [explainer](https://v8.dev/features/weak-references) to learn more about them!
+
+## Other API changes
```

Comment:
Since this is essentially the high-level header from other release PRs, it should be named

```suggestion
## V8 API
```